litmus-portal/backend/self-deployer/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.14 AS builder
+
+ADD . /self-deployer
+WORKDIR /self-deployer
+RUN CGO_ENABLED=0 go build -o /output/deployer -v
```

Comment:
Do we need CGO_ENABLE=0?

Reply:
Yes for using a minimal alpine image for the go binary run I have found CGO_ENABLE=0 is a option that is required, or else some libraries might not work on alpine.

Reply:
CGO is used to compile c code in golang. Try to use it without that

Reply:
I faced issues while doing this and this is one of the common solutions : [StackOverflow](https://stackoverflow.com/questions/36279253/go-compiled-binary-wont-run-in-an-alpine-docker-container-on-ubuntu-host)